installer/bundle/bundle_skel.sh

```diff
@@ -838,9 +879,8 @@ if [ -n "${checkVersionAndCleanUp}" ]; then
 
     # omsconfig
     versionInstalled=`getInstalledVersion omsconfig`
-    versionAvailable=`getVersionNumber $DSC_PKG omsconfig-`
-    if ! python_ctypes_installed > /dev/null 2>&1; then ctypes_text=" (No ctypes)"; fi
-    if shouldInstall_omsconfig; then shouldInstall="Yes"; else shouldInstall="No${ctypes_text}"; fi
+    versionAvailable=`getVersionNumber $DSC_PKG omsconfig-`    
+    if shouldInstall_omsconfig; then shouldInstall="Yes"; else shouldInstall="No (No ctypes)"; fi
```

Comment:
If `shouldInstall_omsconfig` fails, it's not necessarily because of ctypes. I think we should remove the reason text and just print "Yes" or "No"; otherwise we could make a helper function to check ctypes, curl, and gpg and return a reason text from that.